GeoIP2.Benchmark/project.json

```diff
@@ -14,17 +15,20 @@
     "emitEntryPoint": true,
     "keyFile": "../MaxMind.snk"
   },
+
   "dependencies": {
-    "GeoIP2": "2.7.0-*"
+    "GeoIP2": {
+      "target": "project"
+    }
```

Comment:
@oschwald could you please make the same change [here](https://github.com/maxmind/MaxMind-DB-Reader-dotnet/blob/1701f6a/MaxMind.Db.Test/project.json#23) and [here](https://github.com/maxmind/MaxMind-DB-Reader-dotnet/blob/1701f6a/MaxMind.Db.Benchmark/project.json#L17) in `MaxMind-DB-Reader-dotnet`. i.e.

```json
"MaxMind.Db": {
  "target": "project"
},
```

I missed it. This will probably prevent those projects from pulling the lib package from NuGet (and instead use the fresh build from sibling project within the solution).